src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Public.Data/ProcessorClientTests.cs

```diff
@@ -51,6 +53,44 @@ public async Task HttpClientAccepted_ReturnsPublications()
             Assert.Equal(responseBody.InstanceId, right.InstanceId);
         }
 
+        [Fact]
+        public async Task HttpClientBadRequest_ReturnsBadRequest()
+        {
+            _mockHttp.Expect(HttpMethod.Post, Uri.AbsoluteUri)
+                .Respond(
+                    HttpStatusCode.BadRequest,
+                    JsonContent.Create(new ValidationProblemViewModel
+                    {
+                        Errors = new ErrorViewModel[]
+                        {
+                            new() {
+                               Code = Errors.Error1.ToString()
+                            }
+                        }
+                    }));
+
+            var response = await _processorClient.CreateInitialDataSetVersion(releaseFileId: It.IsAny<Guid>());
+
+            _mockHttp.VerifyNoOutstandingExpectation();
+
+            var left = response.AssertLeft();
+            left.AssertValidationProblem(Errors.Error1);
+        }
+
+        [Fact]
+        public async Task HttpClientNotFound_ReturnsNotFound()
+        {
+            _mockHttp.Expect(HttpMethod.Post, Uri.AbsoluteUri)
+                .Respond(HttpStatusCode.NotFound);
+
+            var response = await _processorClient.CreateInitialDataSetVersion(releaseFileId: It.IsAny<Guid>());
```

Comment:
I've not seen `It.IsAny` being used outside of mock Setup or Verify before. Is this correct?